contributors/chairs-and-techleads/process-workflow-guide.md

```diff
@@ -0,0 +1,253 @@
+
+You can find [comprehensive triaging guidelines](https://github.com/kubernetes/community/blob/c78c26294b8bd137dd5cc0c807aadc728dad6a57/contributors/guide/issue-triage.md) here, along with information about triage tools used across the project:
+- **[Gubernator](https://gubernator.k8s.io)**: a personal dashboard that shows you PRs awaiting your review
+- **[Triage Party](https://github.com/google/triage-party)**: a stateless web app to optimize issue and PR triage for large open-source projects using the GitHub API. More details are below. (Cluster Lifecycle, SIG Release) 
+- **[GitHub project boards](https://docs.github.com/en/github/managing-your-work-on-github/managing-project-boards)**: these kanban boards allow you to track GitHub issues and PRs in an automated way. More details are below.
+
+### Triage Meetings
+
+Triaging is highly recommended for improving workflow and responsiveness to contributors. 
+
+Some SIGs, like API Machinery and [Instrumentation](https://docs.google.com/document/d/1FE4AQ8B49fYbKhfg4Tx0cui1V0eI4o3PxoqQPUwNEiU/edit), host triage-specific meetings to manage their backlogs. If your SIG’s backlog is small, however, then you can triage and refine during your bimonthly meetings by setting aside time. In November 2020 we rolled out a needs-triage label to support triage activities. You can add your [user feedback about the label here](https://github.com/kubernetes/community/issues/5299#issuecomment-776914413).
+
+#### Staffing Triage Activities
+
+If you’re a SIG lead who runs triage yourself, you might strongly consider delegating this role to new contributors, including non-code contributors. You might also have contributors who take a strong interest in process and workflow topics—ask and you might be surprised.
+
+SIG Docs rotates the triage facilitator role with its “[wrangler of the week](https://github.com/kubernetes/website/wiki/PR-Wranglers)” schedule. The wrangler balances their time between reviewing PRs and triaging new issues. 
+
+### Triage Party 
+
+#### Example: Triage Party for Skaffold
+
+[Triage Party](https://github.com/google/triage-party) is an open-source tool that focuses on “reducing response latency for incoming GitHub issues and PR's, and ensure[s] that conversations are not lost in the ether.” Project creator Thomas Strömberg reports that using Triage Party for Minikube and Skaffold workflows has led to dramatic improvements in responsiveness to contributor requests. By “dramatic,” he means *40 times faster*.
+
+<img src="triage_party_skaffold.png" width="60%" height="60%">
+
+**Other Uses of Triage Party**:
+- SIG CLI uses it to manage [kubectl](https://tp.kuberneddies.dev/s/daily) workflow, including quarterly bug scrubs and daily/weekly triage
+- SIG Release has set up a Triage Party instance and will begin using it for bug triage, issue triage, and other activities in 1.21.
```

Comment:
Is this live now?

Reply:
Actually yes, it went live a couple of weeks ago: https://release.triage.k8s.io/s/milestone. I'll add that link.